hail/src/main/scala/is/hail/io/bgen/BgenRDDPartitions.scala

```diff
@@ -24,12 +23,11 @@ trait BgenPartition extends Partition {
 
   def contigRecoding: Map[String, String]
 
-  def sHadoopConfBc: Broadcast[SerializableHadoopConfiguration]
+  def bcFS: Broadcast[FS]
 
   def makeInputStream: HadoopFSDataBinaryReader = {
-    val hadoopPath = new Path(path)
-    val fs = hadoopPath.getFileSystem(sHadoopConfBc.value.value)
-    val bfis = new HadoopFSDataBinaryReader(fs.open(hadoopPath))
+    val fileSystem = bcFS.value.fileSystem(path)
+    val bfis = new HadoopFSDataBinaryReader(fileSystem.open(path))
```

Comment:
same as https://github.com/hail-is/hail/pull/6263/files#r290573451

Reply:
same